internal/envoy/install.go

```diff
@@ -36,7 +36,7 @@ func InstallIfNeeded(o *globals.GlobalOpts, p, v string) (string, error) {
 	switch {
 	case os.IsNotExist(err):
 		var ev version.EnvoyVersions // Get version metadata for what we will install
-		ev, err = GetEnvoyVersions(o.EnvoyVersionsURL, o.UserAgent)
+		ev, err = GetEnvoyVersions(o.EnvoyVersionsURL, p, v)
```

Comment:
I realized that we invoke httpGet twice in this path.. here and the inside of untarEnvoy below.. would it be better to call once and reuse the result?

Reply:
it is invoked twice, but different URLs, the tarball is a different hostname in real life (even if in mock we are using the same origin). In any case, it is different URL for "envoy-versions.json" vs "/path/to/tar.xz". Make sense?

Reply:
ah i see, thanks for clarifying!